frontend/projects/upgrade/src/app/features/dashboard/home/components/metrics/metrics.component.ts

```diff
@@ -119,6 +123,18 @@ export class MonitoredMetricsComponent implements OnInit, OnChanges, OnDestroy {
       ]),
     });
 
+    // Subscribe to experiment name changes for reward metric
```

Comment:
The simple object is all that's being emitted to the table in the template, it works fine as far as I can tell without getting form elements involved:

```typescript
if (name) {
    const rewardMetricData = {
        keys: `${name.trim().toUpperCase().replace(/ /g, '_')}_REWARD`,
        operationType: 'Percentage (Success)',
        queryName: 'Success Rate',
    };
    this.rewardMetricDataSource.next([rewardMetricData]);
}
```